src/feedback/FeedbackDialog.tsx

```diff
@@ -127,7 +126,7 @@ export const FeedbackDialog: FunctionComponent<FeedbackDialogProps> = ({
         ].join(" ")}
         style={anchor ? floatingStyles : unanchoredPosition}
       >
-        <FeedbackForm key={featureId} question={title} onSubmit={onSubmit} />
+        <FeedbackForm question={title} onSubmit={onSubmit} />
```

Comment:
@Swiftwork @rrpff Since we removed the featureId here, there was no good information to uniquely identify a key with, so I removed it. Will this have implications on the lifecycle of the widget, animations etc?